mp/src/game/client/c_playerlocaldata.h

```diff
@@ -16,6 +16,12 @@
 #include "mathlib/vector.h"
 #include "playernet_vars.h"
 
+#ifdef CLIENT_DLL
+#define CPostProcessController C_PostProcessController
+#endif
+
+class CPostProcessController;
```

Comment:
Does this need to be in this header? Only the base player uses it, move it there. And use the `C_` name directly, no need for ifdef